sfge/src/main/java/com/salesforce/CliMessagerAppender.java

```diff
@@ -63,9 +64,14 @@ protected CliMessagerAppender(
     public void append(LogEvent event) {
         Level level = event.getLevel();
         if (Level.WARN.equals(level)) {
+            String eventMessage = getEventMessage(event);
+            if (eventMessage.toLowerCase().startsWith("todo:")) {
+                TelemetryUtil.postWarningTelemetry(
```

Comment:
There's a good chance we can find a throwable from event.getThrown(). This may be helpful to store as well.